packages/sdk/src/components/highlighter.ts

```diff
@@ -1,5 +1,12 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import { createHighlighter } from "shiki";
+import { createHighlighterCore } from "shiki/core";
```

Comment:
We now use shiki/core and specific imports instead of importing *all* of shiki for language highlighting.